config/initializers/geocoder.rb

```diff
@@ -15,3 +15,16 @@
   Geocoder.configure(:lookup => :nominatim)
 end
 
+Geocoder::Lookup::Test.add_stub(
```

Comment:
Hi, I moved the stubs to config/environments/test.rb. Just tell me if there's anything more needed to be done.